dvtemplates/build_k8s_cluster.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Build a K8s cluster from kubevirtci
+cd ${PWD}/../kubevirtci
```

Comment:
Please put any strings that have variables in them within double quotes to prevent possible word splitting.

Reply:
Done

Reply:
May just want to put the entire string within the quotes so it's clear it's a single value.
eg: `cd "${PWD}/../kubevirtci"`